src/core/models/answersconfig.js

```diff
@@ -0,0 +1,147 @@
+/** @module AnswersConfig */
+import SearchConfig from '../models/searchconfig';
+import VerticalPagesConfig from '../models/verticalpagesconfig';
+
+/**
+ * AnswersConfig is a model that is used that is used to turn a raw configuration into a data model.
+ * It is used to initialize the Answers SDK
+ */
+export default class AnswersConfig {
+  constructor (rawConfig) {
+    /**
+     * Required, the Yext Answers API Key
+     * @type {string}
+     */
+    this.apiKey = rawConfig.apiKey;
+
+    /**
+     * The internal Yext environment
+     * @type {string}
+     */
+    this.environment = rawConfig.environment;
+
+    /**
+     * Required, the key used for your Answers Experience
+     * @type {string}
+     */
+    this.experienceKey = rawConfig.experienceKey;
+
+    /**
+     *  Optional, initialize components here, invoked when the Answers component library is loaded/ready.
+     *  If components are not added here, they can also be added when the init promise resolves
+     * @type {function()}
+     */
+    this.onReady = rawConfig.onReady;
+
+    /**
+     * Optional*, Yext businessId, *required to send analytics events
+     * @type {number|string|*}
+     */
+    this.businessId = rawConfig.businessId;
+
+    /**
+     * Optional, if false, the library will not fetch pre-made templates. Only use change this to false if you provide a
+     *  template bundle in the `templateBundle` config option or implement custom renders for every component
+     * @type {boolean}
+     */
+    this.useTemplates = rawConfig.useTemplates;
+
+    /**
+     * Optional, additional templates to register with the renderer
+     * @type {Object}
+     */
+    this.templateBundle = rawConfig.templateBundle;
+
+    /**
+     * Optional, provide configuration for each vertical that is shared across components
+     * @type {VerticalPagesConfig}
+     */
+    this.verticalPages = new VerticalPagesConfig(rawConfig.verticalPages);
+
+    /**
+     * Optional, search specific settings
+     * @type {SearchConfig}
+     */
+    this.search = new SearchConfig(rawConfig.search);
+
+    /**
+     * Optional, vertical no results settings. See Vertical No Results
+     * @type {Object}
+     */
+    this.noResults = rawConfig.noResults;
+
+    /**
+     * Optional, the locale will affect how queries are interpreted and the results returned. Defaults to 'en'
+     * @type {string}
+     */
+    this.locale = rawConfig.locale;
```

Comment:
is this defaulting to `en`?

Reply:
it defaults to
```
/** The current locale, injected by the build process */
export const LOCALE = '@@LOCALE';
```
which is located in src/core/constants.js